tools/kagami/src/main.rs

```diff
@@ -13,7 +13,7 @@ use std::{
 
```

Comment:
you should remove `tools/kagami/genesis.json`. I can't comment on empty files on github

Reply:
Oops, I don't know how have I created it